CHANGELOG.md

```diff
@@ -2,6 +2,41 @@
 
 All notable changes to this project will be documented in this file.
 
+## Unreleased - as of Sprint 18 end 2015-01-26
+
+### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+18+Ending+Jan+26%2C+2015%22+label%3Aenhancement)
+
+- Providers
+  - Added refresh status and errors, viewable on Provider Summary Page
+  - OpenStack Host collection and metrics
+  - Amazon EC2:  Added C4 instance types
+- Rest API: VM start, stop, suspend, delete actions added.
+- UI
+ - Multi-character set language support
+ - Cloud Stacks summary and list views
+- Service Dialogs: Dynamic field support for text boxes and text area boxes
+- IPv6:  Pass IPv6 safe address to the socket layer
```

Comment:
Those 2 spaces.  Also this statement is sorta meaningless to a user/reader.  What is the "socket layer"?  Even I don't know what that is.  I feel like this needs better wordikng.